src/main/java/tr/com/srdc/cda2fhir/transform/IValueSetsTransformer.java

```diff
@@ -252,4 +253,12 @@
 	 * @return A value from the FHIR valueset ContactPointSystem.
 	 */
 	ContactPointSystem tTelValue2ContactPointSystem(String cdaTelValue);
+	
+	/**
+	 * Transforms a CDA ProblemStatus code to a AllergyIntoleranceClinicalStatus
+	 * @param code ProblemStatus code
+	 * @return A value from the FHIR AllergyIntoleranceClinicalStatus 
+	 */
+	AllergyIntoleranceClinicalStatus tProblemStatus2AllergyIntoleranceClinicalStatus(String code);
```

Comment:
does it make sense to generalize this function to be tProblemStatus2ClinicalStatus?  The AllergyIntolerance status codes look like a subset of the problem status codes, which we will need to update anyways once we tackle that resource.

Reply:
I will do the conditions next. Can take a look at that point.